docs/paper/admin/reference/configuration/world-configuration.mdx

```diff
@@ -0,0 +1,731 @@
+      start:
+        default: '5'
+        description: 'Days between raid spawns'
+    should-remove-dragon:
+      default: 'false'
+      description: 'Sets whether to remove the dragon if it exists without a portal'
+    spawner-nerfed-mobs-should-jump:
+      default: 'false'
+      description: 'Determines if spawner nerfed mobs should attempt to float (jump)
+        in water'
+    zombie-villager-infection-chance:
+      default: '-1.0'
+      description: 'Sets the chance for villager conversion to zombie villager. Set
+        to -1.0 for default behavior based on game difficulty. Set to 0.0 to always
+        have villagers die when killed by zombies. Set to 100.0 to always convert
+        villagers to zombie villagers'
+    zombies-target-turtle-eggs:
+      default: 'true'
+      description: 'Sets whether zombies and zombified piglins should target turtle
+        eggs. Setting this to false may help with performance, as they won''t search
+        for nearby eggs'
+  entities-target-with-follow-range:
+    default: 'false'
+    description: 'Sets whether the server should use follow range when targeting entities '
+  mob-effects:
+    immune-to-wither-effect:
+      default: 'true'
+      description: 'If the specified entity should be immune to the wither effect'
+    spiders-immune-to-poison-effect:
+      default: 'true'
+      description: 'If spiders should be immune to poison'
+    undead-immune-to-certain-effects:
+      default: 'true'
+      description: 'If undead mobs should be immune to regeneration and poison'
+  spawning:
+    all-chunks-are-slime-chunks:
+      default: 'false'
+      description: 'Instructs the server to treat all chunks like slime chunks, allowing
+        them to spawn in any chunk. This may actually decrease your chances of running
+        into a Slime as they now have a much larger potential spawn area'
+    alt-item-despawn-rate:
+      enabled:
+        default: 'false'
+        description: 'Determines if items will have different despawn rates'
+      items:
+        cobblestone:
+          default: '300'
+          description: 'Sets a custom despawn rate for cobblestone of 300 ticks'
+        <item-type>:
+          default: 'amount'
+          description: 'Determines how long each respective item despawns in ticks.
+            The item ids are the same as those used in the /give command. They can be
+            viewed by enabling advanced item tooltips in-game by pressing **F3 + H**;
+            the item id will appear at the bottom of the tooltip that appears when you
+            hover over an item'
+    count-all-mobs-for-spawning:
+      default: 'false'
+      description: 'Determines whether spawner mobs and other misc mobs are counted
+        towards the global mob limit'
+    creative-arrow-despawn-rate:
+      default: '-1'
+      description: 'The rate, in ticks, at which arrows shot from players in creative
+        mode are despawned'
+    despawn-ranges:
```

Comment:
This section doesn't exists anymore: https://github.com/PaperMC/Paper/blob/master/patches/server/0005-Paper-config-files.patch#L1452. It's now per mob category:
![image](https://github.com/PaperMC/docs/assets/41980282/646b757c-a193-4ff9-9272-d3db3d40d3bd)